### PR TITLE
DAH-2434 Remove name from email layout

### PR DESCRIPTION
**Pull request by jimlin-sfgov**
https://sfgovdt.jira.com/browse/DAH-2434

### CODE REVIEW

File: spec/mailers/emailer_spec.rb [hunk 1]
```diff
@@ -98,7 +98,8 @@
       mail = Emailer.account_update(@record)
 
       VCR.use_cassette('emailer/account_update') do
-        name = 'Hello Jane Doe,'
+        # name = 'Hello Jane Doe,'
```
Comment (tallulahkay) [inline spec/mailers/emailer_spec.rb:101]: Why leave the old functionality in comments in all these tests?

Comment (jimlin-sfgov) [reply spec/mailers/emailer_spec.rb:101]: We'll need them later for the follow up ticket: https://sfgovdt.jira.com/browse/DAH-2471

lmk if you prefer to just remove lines for this kind of situation

Comment (tallulahkay) [reply spec/mailers/emailer_spec.rb:101]: All good to leave them then! Could you include a todo code comment that references that ticket?

Comment (cade-exygy) [reply spec/mailers/emailer_spec.rb:101]: I am okay with the comments in this case

**Review by tallulahkay — APPROVED** [review]
Nice!

**Pull request merged**